test/UserRepository/PdoDatabaseTest.php

```diff
@@ -20,32 +20,44 @@ public function testConstructor()
         $this->assertInstanceOf(UserRepositoryInterface::class, $pdoDatabase);
     }
 
-    public function testAuthenticate()
+    public function getConfig()
     {
-        $pdo = new PDO('sqlite:'. __DIR__ . '/../TestAssets/pdo.sqlite');
-        $pdoDatabase = new PdoDatabase($pdo, [
+        return [
             'table' => 'user',
             'field' => [
-                'username' => 'username',
-                'password' => 'password',
+                'identity' => 'username',
+                'password' => 'password'
             ]
-        ]);
+        ];
+    }
+
+    public function testAuthenticate()
+    {
+        $pdo = new PDO('sqlite:'. __DIR__ . '/../TestAssets/pdo.sqlite');
+        $pdoDatabase = new PdoDatabase($pdo, $this->getConfig());
 
         $user = $pdoDatabase->authenticate('test', 'password');
         $this->assertInstanceOf(UserInterface::class, $user);
-        $this->assertEquals('test', $user->getUsername());
+        $this->assertEquals('test', $user->getIdentity());
+    }
+
+    /**
+     * @expectedException Zend\Expressive\Authentication\Exception\RuntimeException
```

Comment:
I would suggest to use method `expectException` just before line where the exception should be thrown, otherwise it could be thrown in any line of this test...

Reply:
Good catch, thanks!